CONTRIBUTING.md

```diff
@@ -25,13 +25,14 @@ To contribute to this project, it is recommended that you follow these steps:
       1. Install Python from https://www.python.org/downloads/ . 
       2. Add Python install location to the Windows path.
       3. Install PyPI, following the instructions on https://pip.pypa.io/en/latest/installing/ .
-      4. Install Wheel using PyPI.
-      5. Install Mock using PyPI.
-      6. Install Mako using PyPI.
-      7. Install mingw (msys-base) following the instructions on http://www.mingw.org/wiki/Getting_Started/ . 
-      8. Add <mingw Install Path>\msys\1.0\bin to Windows path.
-      9. Call "make clean".
-      10. Call "make".
+      4. Install required Python Modules using PyPI.
+         - wheel (https://pypi.python.org/pypi/wheel/) .
+         - mock (https://pypi.python.org/pypi/mock/) .
+         - mako (https://pypi.python.org/pypi/mako/) .
+      5. Install mingw (msys-base) following the instructions on http://www.mingw.org/wiki/Getting_Started/ . 
+      6. Add <mingw Install Path>\msys\1.0\bin to Windows path.
+      7. Call "make clean".
+      8. Call "make".
```

Comment:
FYI: this is Markdown so you can have it link without putting the URL in the reader's face.
https://guides.github.com/features/mastering-markdown/